src/main/java/com/iexec/worker/tee/scone/LasServicesManager.java

```diff
@@ -0,0 +1,88 @@
+package com.iexec.worker.tee.scone;
+
+import com.iexec.sms.api.SmsClient;
+import com.iexec.sms.api.SmsClientProvider;
+import com.iexec.sms.api.TeeWorkflowConfiguration;
+import com.iexec.worker.config.WorkerConfigurationService;
+import com.iexec.worker.docker.DockerService;
+import com.iexec.worker.sgx.SgxService;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PreDestroy;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class LasServicesManager {
+    private final SconeConfiguration sconeConfiguration;
+    private final SmsClientProvider smsClientProvider;
+    private final WorkerConfigurationService workerConfigService;
+    private final SgxService sgxService;
+    private final DockerService dockerService;
+
+    //TODO: Purge entry when task is over (completed/failed)
+    private final Map<String, LasService> chainTaskIdToLasService = new HashMap<>();
+    private final Map<String, LasService> lasImageUriToLasService = new HashMap<>();
+
+    public LasServicesManager(SconeConfiguration sconeConfiguration,
+                              SmsClientProvider smsClientProvider,
+                              WorkerConfigurationService workerConfigService,
+                              SgxService sgxService,
+                              DockerService dockerService) {
+        this.sconeConfiguration = sconeConfiguration;
+        this.smsClientProvider = smsClientProvider;
+        this.workerConfigService = workerConfigService;
+        this.sgxService = sgxService;
+        this.dockerService = dockerService;
+    }
+
+    public boolean startLasService(String chainTaskId) {
+        // SMS client should already have been created once before.
+        // If it couldn't be created, then the task would have been aborted.
+        // So the following won't throw an exception.
+        final SmsClient smsClient = smsClientProvider.getOrCreateSmsClientForTask(chainTaskId);
+
+        final TeeWorkflowConfiguration config = smsClient.getTeeWorkflowConfiguration();
+        if (config == null) {
+            throw new RuntimeException("Missing tee workflow configuration");
```

Comment:
This exception does not seem to be caught in the code.
The exception will go up to `start` and `compute` methods in `TaskManagerService` where it will never be caught.
Do we have a potential point where a task lifecycle could be broken ?
Wouldn't it be a safer implementation with a log message and a returned `false` literal ?